Makefile

```diff
@@ -0,0 +1,87 @@
+.PHONY: build-garage-ci build-garage-headless build-garage-nvidia \
```

Comment:
perhaps you can remove `-garage-` from all these directives, since it's obvious they are garage

`run-nvidia`, etc.

Reply:
Done